tests/test_sign_cmd.py

```diff
@@ -31,7 +31,15 @@ def test_sign_tx_short_tx(firmware, backend, navigator, test_name):
         to="de0b295669a9fd93d5f28d9ec85e40f4cb697bae",
         memo="For u EthDev"
     ).serialize()
-
+    
+    # Enable display of transaction memo (NBGL devices only)
+    if not firmware.device.startswith("nano"):
+        navigator.navigate([NavInsID.USE_CASE_HOME_SETTINGS,
+                            NavIns(NavInsID.TOUCH, (200, 113)),
+                            NavInsID.USE_CASE_SUB_SETTINGS_EXIT],
+                            screen_change_before_first_instruction=False,
+                            screen_change_after_last_instruction=False)
+    
     # Send the sign device instruction.
     # As it requires on-screen validation, the function is asynchronous.
```

Comment:
please use the same tests as for C BP (especially the scenarios feature)

Reply:
Done.